### PR TITLE
ansible: add auto start nebd-server at boot

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by SeanHai** [comment]
recheck

File: curve-ansible/roles/clean/tasks/include/clean_nebd.yml [hunk 1]
```diff
@@ -21,3 +21,18 @@
...
+  block:
+    - name: get distro name
+      vars:
+        distro:
+      include_tasks: common_tasks/get_distro_name.yml
+    - name: disable on debain ubuntu
+      shell: sudo update-rc.d -f nebd-daemon remove
+      when: "'Ubuntu' in distro or 'Debian' in distro"
+    - name: disable on centos
+      shell: sudo chkconfig nebd-daemon off &&  sudo chkconfig --del nebd-daemon
```
Comment (Wine93) [inline curve-ansible/roles/clean/tasks/include/clean_nebd.yml:35]: redundant blank space.

File: curve-ansible/roles/generate_config/templates/tools.conf.j2 [hunk 2]
```diff
@@ -20,13 +20,15 @@ rpcRetryTimes={{ tool_rpc_retry_times }}
   {% set _ = etcd_address.append("%s:%s" % (etcd_ip, etcd_port)) -%}
 {% endfor -%}
 etcdAddr={{ etcd_address | join(',') }}
+{% if 'snapshotclone' in group_names and ansible_fqdn in groups['snapshotclone'] -%}
```
Comment (cw123) [inline curve-ansible/roles/generate_config/templates/tools.conf.j2:23]: What's the difference between ansible_fqdn and ansible_hostname?

File: curve-ansible/roles/clean/tasks/include/clean_nebd.yml (hunk 1 above)
Comment (SeanHai) [reply curve-ansible/roles/clean/tasks/include/clean_nebd.yml:35]: > redundant blank space.

done

Comment (SeanHai) [reply curve-ansible/roles/clean/tasks/include/clean_nebd.yml]: > redundant blank space.

done

File: curve-ansible/roles/stop_service/tasks/include/stop_nebd.yml [hunk 3]
```diff
@@ -26,5 +26,5 @@
 
 - name: stop nebd without daemon
   ignore_errors: true
-  shell: killall {{ curve_bin_dir }}/nebd-server
+  shell: sudo killall nebd-server
```
Comment (cw123) [inline curve-ansible/roles/stop_service/tasks/include/stop_nebd.yml:29]: Does this sudo need {{ sudo }} instead?

Comment (SeanHai) [reply curve-ansible/roles/stop_service/tasks/include/stop_nebd.yml:29]: > Does this sudo need {{ sudo }} instead?

done

**Review by cw123 — APPROVED** [review]

**Review by Wine93 — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**